README.md

```diff
@@ -102,6 +102,8 @@ a double dash to prevent input files being used as option arguments:
                                 sequences.
     --config-file <file>        Read `minify()` options from JSON file.
     -d, --define <expr>[=value] Global definitions.
+    --ecma <version>            Specifiy ECMAScript release number: 5, 6, 7 or 8.
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/2157/commits/33de720c40acced40fc95cdd7a3b004b6c1ac332